### PR TITLE
feat(create landing page

### PR DESCRIPTION
**Pull request by xcixor**
-Create the folder structure for the application

-Implement the landing page markup

-Add responsiveness to the page

-Style the page

Add images to enhance attractiveness

[Delivers #158438713]

### CODE REVIEW

**Comment by xcixor** [comment]
#### #158438713 Creating the landing page

#### Purpose of the PR
-Creates the landing page
#### Tasks to be completed
- Design the landing page
#### How should this be manually tested?
* Clone the repo like so:
* #### $ git clone https://github.com/xcixor/my-ride.git
* navigate to the UI folder 
* Open the index file with your favorite browser
* To preview on mobile in chrome inspect the page and the browser's inspection tool will provide previews for various screens
#### Background information
No framework has been used to create the app
#### What are the relevant pivotal tracker stories?
#158438713 - The client should be able to create awareness for their application

#### Screenshots 
screenshot from 2018-06-19 19-23-00
screenshot from 2018-06-19 19-23-54
screenshot from 2018-06-19 19-24-14
screenshot from 2018-06-19 19-24-52
screenshot from 2018-06-19 19-25-13

File: UI/static/css/index.css [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    opacity: 0.8;
+    display: flex;
+    align-items: flex-end;
+}
+
+.attention-call {
+    grid-area: attention-call;
+    justify-self: center;
+    width: 100%;
+    /* position: absolute; */
```
Comment (JoshuaOndieki) [inline UI/static/css/index.css:33]: Remove commented out code before committing

**Review by JoshuaOndieki — CHANGES_REQUESTED** [review]

File: UI/static/css/index.css (hunk 1 above)
Comment (evamaina) [reply UI/static/css/index.css:33]: wonderful UI

Comment (artorious) [reply UI/static/css/index.css:33]: Great UI. Looks good

My only concern is related to version control, I've noticed that you've merged into master.
Check this out [Feature-branch workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)

Comment (evamaina) [reply UI/static/css/index.css:33]: will check, thanx

**Pull request merged**